filesystem/src/main/java/org/frankframework/filesystem/AbstractFileSystemListener.java

```diff
@@ -140,13 +141,22 @@ public void configure() throws ConfigurationException {
 		if (getNumberOfBackups()>0 && !(fileSystem instanceof IWritableFileSystem)) {
 			throw new ConfigurationException("FileSystem ["+ClassUtils.nameOf(fileSystem)+"] does not support setting attribute 'numberOfBackups'");
 		}
+		if (isFileTimeSensitive() && !(fileSystem instanceof IWritableFileSystem)) {
+			throw new ConfigurationException("FileSystem ["+ClassUtils.nameOf(fileSystem)+"] does not support setting attribute 'fileTimeSensitive'");
```

Comment:
There was technically not an error, but this was really meant to be used with "writeable" filesystems because it involved renaming files.